src/PrestaShopBundle/Controller/Admin/AttributeController.php

```diff
@@ -174,7 +174,7 @@ public function ensureProductHasDefaultCombination(Product $product, array $comb
         if (count($combinations)) {
             $defaultProductAttributeId = $product->getDefaultIdProductAttribute();
             if (!$defaultProductAttributeId) {
-                list(, $firstAttributeCombination) = each($combinations[0]);
+                $firstAttributeCombination = reset($combinations[0]);
```

Comment:
Same comment here, I'm not sure get the same result.

Reply:
same answer ^^